### PR TITLE
Compatible ID29

### PR DESCRIPTION
**Pull request by kirya-dev**

### CODE REVIEW

File: udsoncan/j2534.py [hunk 1]
```diff
@@ -243,12 +256,18 @@ def PassThruReadVersion(self, DeviceID):
...
+        result = dllPassThruGetLastError(pErrorDescription)
 
+        return Error_ID(hex(result)), str(pErrorDescription.value)
+
+    def PassThruIoctl(self, Handle, IoctlID, ioctlInput=None, ioctlOutput=None):
         if ioctlInput is None:
             pInput = POINTER(c_ulong)()
         else:
-            pInput = c_ulong(ioctlInput.value)
+            # pInput = c_ulong(ioctlInput.value)
```
Comment (kirya-dev) [inline udsoncan/j2534.py:269]: This comment becouse incompatible with passing Ioctl_Flags.TX_IOCTL_SET_DLL_DEBUG_FLAGS

**Comment by pylessard** [comment]
@joeFischetti  : As the original author of the j2534 extension, I would love to have your feedback.
I do not know J2534 myself, never used it.

**Comment by pylessard** [comment]
@kirya-dev  : I see no issue with your code, but my knowledge of J2534 is limited. How thorough  were your test?
Do you think it is feasible to add unit test for that module?

**Comment by kirya-dev** [comment]
I think it possible, but I not have such experience in python tests 🤔
You can push scaffolding for it. I fix it for that task.

I think we need write any layer between DLL & J2534 class

**Pull request merged**